### ISSUE TITLE
Cannot use esc to exit "symbols" dialog without using "tab" first

### ISSUE THREAD

**Issue by brooklynlash** [phetsims/fourier-making-waves#199]
**Test device**
Chromebook

**Browser**
Chrome

**Problem description**
For https://github.com/phetsims/qa/issues/711
Inconsistent with other dialogs, you cannot immediately click "esc" to exit the Symbols dialog without using "tab" to highlight it first. For example, the keyboard shortcuts menu automatically highlights the "x" so you can click "esc" without any extra steps. 

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Fourier: Making Waves‬
URL: https://phet-dev.colorado.edu/html/fourier-making-waves/1.0.0-rc.1/phet/fourier-making-waves_all_phet.html
Version: 1.0.0-rc.1 2021-09-28 15:44:23 UTC
Features missing: applicationcache, generatedcontent, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (X11; CrOS x86_64 13904.97.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.167 Safari/537.36
Language: en-US
Window: 1536x714
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 32 uniform: 4096
Texture: size: 16384 imageUnits: 32 (vertex: 32, combined: 192)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by pixelzoom** [comment]
Thanks @brooklynlash.  This is a duplicate of https://github.com/phetsims/fourier-making-waves/issues/194.  All dialogs are being fixed in that issue.
